Makefile

```diff
@@ -132,3 +133,17 @@ test-e2e-local: $(KIND) $(YQ) $(GINKGO)
 .PHONY: oidc-up
 oidc-up: $(KIND) $(YQ)
 	@$(REPO_ROOT)/hack/oidc-up.sh
+
+# speed-up skaffold deployments by building all images concurrently
+export SKAFFOLD_BUILD_CONCURRENCY = 0
+extension-up: export SKAFFOLD_DEFAULT_REPO = garden.local.gardener.cloud:5001
```

Comment:
Should we switch to `registry.local.gardener.cloud:5000`?

If yes: we should also do so in the `skaffold.yaml` -> `insecureRegistries`

Reply:
Switched to `registry.local.gardener.cloud:5001`, the port is about to be changed from 5000 to 5001 with https://github.com/gardener/gardener/pull/13661, I will not be trying to cover the case where the port is still 5000.